aleph/index/entities.py

```diff
@@ -64,6 +64,8 @@ def iter_entities(
     excludes=None,
     filters=None,
     sort=None,
+    es_scroll="5m",
```

Comment:
Do you suppose, for consistency, we could reference values set in `SETTINGS` here as well? Perhaps have `ES_SCROLL` and `ES_XREF_SCROLL`?